lib/ui/pointer.dart

```diff
@@ -61,7 +89,25 @@ enum PointerDeviceKind {
   /// A pointer device with a stylus that has been inverted.
   invertedStylus,
 
-  /// A touch-based pointer device with an indirect surface.
+  /// Gestures from a touch-based pointer device with an indirect surface.
+  ///
+  /// On supporting platforms, when the user is operating on a physical
+  /// trackpad, events with kind [trackpad] are dispatched when the user
+  /// performs gestures on the trackpad, such as panning, zooming, scrolling,
+  /// or rotating. The [trackpad] kind is only found with [PointerChange] `add`,
```

Comment:
I don't know anything about the class being documented here but: I can't parse the sentence that begins "The [trackpad] kind ..."

Reply:
I've made it into an independent paragraph and rephrased it as follows:
```dart
  /// Events with kind [trackpad] can only have a [PointerChange] of `add`,
  /// `remove`, and pan-zoom related values.
```
Does it make more sense now?